src/Selenium2Library/keywords/_element.py

```diff
@@ -1,14 +1,32 @@
 from selenium.webdriver.common.keys import Keys
 from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.remote.webelement import WebElement
 from Selenium2Library import utils
 from Selenium2Library.locators import ElementFinder
 from keywordgroup import KeywordGroup
 
+try:
+    basestring  # attempt to evaluate basestring
+    def isstr(s):
+        return isinstance(s, basestring)
+except NameError:
+    def isstr(s):
+        return isinstance(s, str)
+
 class _ElementKeywords(KeywordGroup):
 
     def __init__(self):
         self._element_finder = ElementFinder()
 
+    # Public, get element(s)
+
+    def get_webelements(self, locator):
```

Comment:
Instead of ```Get WebElements```, how would you feel about just ```Get Elements``` . Also, would it be useful to have a ```Get Element``` that only gets a single element, or would that be unnecessary/confusing?